### PR TITLE
Air compressor, Hydraulic press & Water purifier appliances

### PR DESCRIPTION
**Pull request by ZombieZilla**
#### Summary
Content "Air compressor, water purifier and hydraulic press converted to appliances"

#### Purpose of change
Finishing off a few more easy appliances before content freeze

#### Testing
All appliances work in every function

### CODE REVIEW

File: data/json/furniture_and_terrain/appliances.json [hunk 1]
```diff
@@ -91,7 +91,7 @@
     "id": "ap_oven",
     "name": { "str": "oven" },
     "categories": [ "utility" ],
-    "description": "A standard electric convection-based oven, commonly used for heating and cooking food.",
+    "description": "This device effectively sterilizes water.",
```
Comment (I-am-Erk) [inline data/json/furniture_and_terrain/appliances.json:94]: You seem to have swapped these descriptions

File: data/json/furniture_and_terrain/furniture-plumbing.json [hunk 2]
```diff
@@ -135,12 +135,13 @@
     "id": "f_water_purifier",
     "looks_like": "f_water_heater",
     "name": "water purifier",
-    "description": "This device effectively sterilizes water, though without power source, like UPS, it's only good for parts.",
+    "description": "This device effectively sterilizes water, examine to turn into an appliance.",
```
Comment (I-am-Erk) [inline data/json/furniture_and_terrain/furniture-plumbing.json:138]:
```suggestion
    "description": "This device effectively sterilizes water. You can plug it in to your own power grid to use it.",
```

**Comment by I-am-Erk** [comment]
I don't love the "examine to turn into an appliance" phrase, as it is entirely using back-end terminology and doesn't mean much to a new player

**Comment by ZombieZilla** [comment]
> I don't love the "examine to turn into an appliance" phrase, as it is entirely using back-end terminology and doesn't mean much to a new player

Changed to "You can take this down and plug it into your own power grid to use it", should be more clear to players now

**Pull request merged**